quality-metrics/qm-pull-requests.md

```diff
@@ -0,0 +1,25 @@
+---
+Author: John Bellone <jbellone@bloomberg>
+Status: Draft
+License: Apache 2.0
+---
+
+## Cookbook has closed pull-requests on GitHub.
```

Comment:
Maybe it makes more sense for this to be number of *issues* since cookbooks can have an [issues_url](https://docs.chef.io/cookbook_repo.html#Settings)?